network-api/networkapi/wagtailpages/pagemodels/blog/blog.py

```diff
@@ -118,6 +118,9 @@ class Meta:
 
 
 class BlogPage(FoundationMetadataPageMixin, Page):
+    # Custom base form for additional validation
+    base_form_class = BlogPageForm
+
```

Comment:
**Kudos** Nice, I think this is a good idea to move configuration like this above the fields 👍